hackathon_site/registration/jinja2/registration/emails/accepted_email_body.html

```diff
@@ -0,0 +1,10 @@
+<p>Hello {{ user.first_name|striptags }},</p>
+
+<p>The {{ hackathon_name }} team has reviewed your application, and we’re excited to welcome you to {{hackathon_name}} 2020! 
```

Comment:
Minor, but for consistency pad the inside of the `{{ }}` blocks with spaces (in reference to the last `{{hackathon_name}}` here, should be `{{ hackathon_name }}`. Won't comment on all of them, but go through all these templates to catch the few without spaces.

Reply:
Also shouldn't have the 2020 hard coded into the template, see my other comment.